src/OrbitGl/SamplingReportDataView.h

```diff
@@ -36,7 +36,10 @@ class SamplingReportDataView : public DataView {
   void OnRefresh(const std::vector<int>& visible_selected_indices,
                  const RefreshMode& mode) override;
 
-  void UpdateSelectedAddressesAndTid(const std::vector<int>& indices);
+  void UpdateSelectedIndicesAndFunctionIds(const std::vector<int>& selected_indices);
+  void UpdateSelectedIndicesOnFunctionsChanged();
+  // The callstack view will be updated according to the visible selected addresses and thread id.
+  void UpdateVisibleSelectedAddressesAndTid(const std::vector<int>& visible_selected_indices);
```

Comment:
I think these can be private

Reply:
Ah, right! Thanks for pointing out this. Changed accordingly.